lib/components/narrative/metro/metro-itinerary.tsx

```diff
@@ -245,6 +250,17 @@ class MetroItinerary<Props> extends NarrativeItinerary {
       currency
     )
 
+    Object.keys(accessibilityScoreGradationMap).forEach((key) => {
```

Comment:
The gradation map fix is missing here. Also I'd do this instead as it's cleaner
```suggestion
    Object.keys(accessibilityScoreGradationMap || {}).forEach((key) => {
```